include/avif/avif.h

```diff
@@ -1156,6 +1269,9 @@ typedef struct avifEncoder
     // changeable encoder settings
     int quality;
     int qualityAlpha;
+#if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
+    int qualityGainMap;
+#endif
```

Comment:
In anticipation of the libavif 1.0.0 release, the new member `qualityGainMap` should be added to the end of the `avifEncoder` struct, after all the current struct members.